crates/ads126x/src/register.rs

```diff
@@ -0,0 +1,94 @@
+use bitflags::bitflags;
+
+#[repr(u8)]
+pub enum Register {
+    ID = 0x00,
+    POWER = 0x01,
+    INTERFACE = 0x02,
+    MODE0 = 0x03,
+    MODE1 = 0x04,
+    MODE2 = 0x05,
+    INPMUX = 0x06,
+    OFCAL0 = 0x07,
+    OFCAL1 = 0x08,
+    OFCAL2 = 0x09,
+    FSCAL0 = 0x0A,
+    FSCAL1 = 0x0B,
+    FSCAL2 = 0x0C,
+    IDACMUX = 0x0D,
+    IDACMAG = 0x0E,
+    REFMUX = 0x0F,
+    TDACP = 0x10,
+    TDACN = 0x11,
+    GPIOCON = 0x12,
+    GPIODIR = 0x13,
+    GPIODAT = 0x14,
+    ADC2CFG = 0x15,
+    ADC2MUX = 0x16,
+    ADC2OFC0 = 0x17,
+    ADC2OFC1 = 0x18,
+    ADC2FSC0 = 0x19,
+    ADC2FSC1 = 0x1A,
+}
+
+bitflags! {
+    pub struct IdRegister: u8 {
+        const _ = !0; // Source may set any bits
+    }
+}
+
+pub enum DevId {
+    ADS1262 = 0b000,
+    ADS1263 = 0b001,
+}
+
+impl IdRegister {
+    pub fn get_rev_id(&self) -> u8 {
+        self.bits() & 0b0001_1111
+    }
+
+    pub fn get_dev_id(&self) -> DevId {
```

Comment:
We could also return a `u8` here if that's preferable, though the enum might make more sense here given it represents different boards.

Reply:
I like the enum, let's keep it.